deploy/backplane/acs/01-acs-admins-cluster-aggregate.ClusterRole.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-cluster-aggregate
+  labels:
+    managed.openshift.io/aggregate-to-acs-admins: "cluster"
+rules:
+# ACS SRE can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+# ACS SRE can get infrastructure details (cluster)
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
+  - oauths
+  verbs:
+  - 'get'
+# ACS SRE can manage namespaces / projects
+- apiGroups:
+  - project.openshift.io
+  resources:
+  - projects
+  verbs:
+  - create
```

Comment:
`delete` permission is not included, can create without ability to remove.
That being said, why is create project and namespace required?
Is it something needed daily or can it be break-glass?

Reply:
We do not need permissions to create/patch/write projects, @Nanyte25 this can be removed